HopsanGUI/Dialogs/OptimizationScriptWizard.h

```diff
@@ -90,12 +102,12 @@ private slots:
     void generateParameterCode(QString &templateCode);
     void generateCommonOptions(QString &templateCode);
     QString generateFunctionCode(int i);
-    bool verifyNumberOfVariables(int i, int nSelVar);
+    bool verifyNumberOfVariables(int i, int nSelVar, bool noWarning);
     bool loadObjectiveFunctions();
 
     void loadConfiguration();
 
-    void addObjectiveFunction(int idx, double weight, double norm, double exp, QList<QStringList> selectedVariables, QStringList objData);
+    void addObjectiveFunction(int idx, double weight, double norm, double exp, QList<QStringList> selectedVariables, QStringList objData, bool noWarning=false);
```

Comment:
Double negation is confusing
I have done this sometimes as well, but we should stop,
printWarning=true would feel better.

Reply:
Also in other places